ironfish-rust-nodejs/tsconfig.tsbuildinfo

```diff
@@ -0,0 +1 @@
,[93,96],[54,59,65,67,76,82,92,93,95,97],[82,93,98],[93,140],[65,68,70,73,82,89,92,93,98,100],[93,145],[93,111,112],[93,111,112,113,114],[93,116]],"referencedMap":[[41,1],[42,2],[46,3],[44,1],[43,1],[49,4],[45,3],[47,5],[48,3],[101,6],[102,1],[103,1],[105,7],[106,8],[107,1],[108,1],[109,9],[110,10],[118,11],[119,1],[120,1],[121,12],[122,13],[124,14],[125,15],[123,16],[126,17],[127,18],[128,19],[129,20],[130,21],[131,22],[132,23],[133,24],[134,25],[135,26],[104,1],[136,1],[50,27],[51,27],[53,28],[54,29],[55,30],[56,31],[57,32],[58,33],[59,34],[60,35],[61,36],[62,37],[63,37],[64,38],[65,39],[66,40],[67,41],[52,1],[99,1],[68,42],[69,43],[70,44],[100,45],[71,46],[72,47],[73,48],[74,49],[75,50],[76,51],[77,52],[78,53],[79,54],[80,55],[81,56],[82,57],[84,58],[83,59],[85,60],[86,61],[87,62],[88,63],[89,64],[90,65],[91,66],[92,67],[93,68],[94,69],[95,70],[96,71],[97,72],[98,73],[137,1],[138,1],[139,1],[141,74],[140,1],[142,1],[143,1],[144,75],[145,1],[146,76],[147,1],[111,1],[113,77],[115,78],[114,77],[112,1],[117,79],[116,1],[8,1],[9,1],[11,1],[10,1],[2,1],[12,1],[13,1],[14,1],[15,1],[16,1],[17,1],[18,1],[19,1],[3,1],[4,1],[23,1],[20,1],[21,1],[22,1],[24,1],[25,1],[26,1],[5,1],[27,1],[28,1],[29,1],[30,1],[6,1],[31,1],[32,1],[33,1],[34,1],[7,1],[39,1],[35,1],[36,1],[37,1],[38,1],[1,1],[40,1]],"exportedModulesMap":[[41,1],[42,2],[46,3],[44,1],[43,1],[49,4],[45,3],[47,5],[48,3],[101,6],[102,1],[103,1],[105,7],[106,8],[107,1],[108,1],[109,9],[110,10],[118,11],[119,1],[120,1],[121,12],[122,13],[124,14],[125,15],[123,16],[126,17],[127,18],[128,19],[129,20],[130,21],[131,22],[132,23],[133,24],[134,25],[135,26],[104,1],[136,1],[50,27],[51,27],[53,28],[54,29],[55,30],[56,31],[57,32],[58,33],[59,34],[60,35],[61,36],[62,37],[63,37],[64,38],[65,39],[66,40],[67,41],[52,1],[99,1],[68,42],[69,43],[70,44],[100,45],[71,46],[72,47],[73,48],[74,49],[75,50],[76,51],[77,52],[78,53],[79,54],[80,55],[81,56],[82,57],[84,58],[83,59],[85,60],[86,61],[87,62],[88,63],[89,64],[90,65],[91,66],[92,67],[93,68],[94,69],[95,70],[96,71],[97,72],[98,73],[137,1],[138,1],[139,1],[141,74],[140,1],[142,1],[143,1],[144,75],[145,1],[146,76],[147,1],[111,1],[113,77],[115,78],[114,77],[112,1],[117,79],[116,1],[8,1],[9,1],[11,1],[10,1],[2,1],[12,1],[13,1],[14,1],[15,1],[16,1],[17,1],[18,1],[19,1],[3,1],[4,1],[23,1],[20,1],[21,1],[22,1],[24,1],[25,1],[26,1],[5,1],[27,1],[28,1],[29,1],[30,1],[6,1],[31,1],[32,1],[33,1],[34,1],[7,1],[39,1],[35,1],[36,1],[37,1],[38,1],[1,1],[40,1]],"semanticDiagnosticsPerFile":[41,42,46,44,43,49,45,47,48,101,102,103,105,106,107,108,109,110,118,119,120,121,122,124,125,123,126,127,128,129,130,131,132,133,134,135,104,136,50,51,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,52,99,68,69,70,100,71,72,73,74,75,76,77,78,79,80,81,82,84,83,85,86,87,88,89,90,91,92,93,94,95,96,97,98,137,138,139,141,140,142,143,144,145,146,147,111,113,115,114,112,117,116,8,9,11,10,2,12,13,14,15,16,17,18,19,3,4,23,20,21,22,24,25,26,5,27,28,29,30,6,31,32,33,34,7,39,35,36,37,38,1,40]},"version":"4.3.4"}
```

Comment:
Not related to these changes, but this might need to be added to a gitignore, it shouldn't be committed.

Reply:
Added this: https://github.com/iron-fish/ironfish/pull/439/commits/ad845be177128d7464cba954f5f9643b0475b370#diff-f0e9030b6607a25b729aa5d68b2556d39b0472034d994661dafdffb3570efa85R6